### ISSUE TITLE
Incorporate feedback into docs

### ISSUE THREAD

**Issue by 00vareladavid** [JuliaLang/Pkg.jl#1311]
There seems to be plenty of good feedback here: https://discourse.julialang.org/t/package-manager-documentation/27351.

Just making sure that discussion doesn't get lost.

**Comment by StefanKarpinski** [comment]
Yes, I've had that bookmarked and open in my browser since last week. Definitely excellent feedback to incorporate.
